scripts/traccc_throughput_mt_profiler.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+#
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2023 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+#
+# Simple script running the selected instance of the multi-threaded throughput
+# executable on a whole set of "measurement points".
+#
+
+# Stop on errors.
+set -e
+
+# Function printing the usage information for the script.
+usage() {
+   echo "Script running a suite of multi-threaded throughput tests"
+   echo ""
+   echo "Usage: traccc_throughput_mt_profiling.sh [options]"
+   echo ""
+   echo "Basic options:"
+   echo "  -x <executable>      Selects the executable to use"
+   echo "  -t <maxThreads>      Maximum number of threads to test"
+   echo "  -r <repetitions>     The number of repetitions in the test"
+   echo "  -e <eventMultiplier> Multiplier for the number of events per thread"
+   echo "  -c <csvFile>         Name of the output CSV file"
+   echo "  -h                   Print this help"
+   echo ""
+   echo "Detailed options:"
+   echo "  -g <detectorGeomFile> Detector geometry CSV file"
+   echo "  -d <digiConfigFile>   Detector digitization JSON file"
+   echo ""
+}
+
+# Parse the command line arguments.
+TRACCC_EXECUTABLE="traccc_throughput_mt"
```

Comment:
Let's allow the user to set these as environment variables and only use these defaults if nothing is set.

```suggestion
TRACCC_EXECUTABLE=${TRACCC_EXECUTABLE:-"traccc_throughput_mt}"
```

Reply:
Sure. That could help in some situations.